server/src/main/java/org/jboss/as/server/parsing/StandaloneXml.java

```diff
@@ -528,16 +527,16 @@ private void parseHttpManagementInterface1_0(final XMLExtendedStreamReader reade
                 final Attribute attribute = Attribute.forName(reader.getAttributeLocalName(i));
                 switch (attribute) {
                     case INTERFACE: {
-                        HttpManagementResourceDefinition.INTERFACE.parseAndSetParameter(value, mgmtSocket, reader);
+                        //ignore
                         hasInterfaceName = true;
```

Comment:
Yes, please fail. Ignoring either means a failure later with a less useful error message, or somehow it doesn't fail but the server isn't manageable.